CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+2.4.0
+-----
+
+* [BC BREAK] The `@Route` annotation and all its children no longer extend SensioFrameworkExtraBundle's annotation.
+  The main effect is that `@Route::$service` is no longer available.
```

Comment:
We should mention that you should use FCQN as service ids instead. And we should also update the branch alias in the `composer.json` file.

Reply:
Thanks, sorry I forgot to take care of this :)